presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/thrift/BridgingHiveMetastore.java

```diff
@@ -186,6 +187,24 @@ public void renameTable(String databaseName, String tableName, String newDatabas
         alterTable(databaseName, tableName, table);
     }
 
+    @Override
+    public void commentTable(String databaseName, String tableName, Optional<String> comment)
+    {
+        Optional<org.apache.hadoop.hive.metastore.api.Table> source = delegate.getTable(databaseName, tableName);
+        if (!source.isPresent()) {
+            throw new TableNotFoundException(new SchemaTableName(databaseName, tableName));
+        }
+        org.apache.hadoop.hive.metastore.api.Table table = source.get();
+
+        Map<String, String> parameters = table.getParameters().entrySet().stream()
+                .filter(entry -> !entry.getKey().equals(TABLE_COMMENT))
+                .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue));
+        comment.ifPresent(value -> parameters.put(TABLE_COMMENT, comment.get()));
+
+        table.setParameters(ImmutableMap.copyOf(parameters));
```

Comment:
Why ImmutableMap.copyOf?

Reply:
Removed. It was debris during development.